source/unified-test-format/tests/valid-pass/entity-runCursorCommand.yml

```diff
@@ -0,0 +1,115 @@
+description: runCursorCommand
+schemaVersion: '1.3'
```

Comment:
I'll note that adding the `commandCursor` entity type and `createCommandCursor` operation will not warrant a bump to the schema version.

Reply:
good note, I added the entity type, but no new schema version